dts/arm/nuvoton/npcx/npcx7-psl-ctrl-map.dtsi

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	def_psl_conf_list {
+		compatible = "nuvoton,npcx-pslctrl-conf";
+
+		/* PSL input IO configurations */
+		psl_in1: psl_iod2 { offset = <0>; /* PSL_IN1/PIND2 */
```

Comment:
Please use normal style here, with nothing after {, and } on its own line

Reply:
Done.